crates/wash-lib/tests/parser/files/rust_actor_with_revision.toml

```diff
@@ -0,0 +1,18 @@
+language = "rust"
+type = "actor"
+name = "testactor"
+version = "0.1.0"
+revision = 666
```

Comment:
Suggestion here: I think we shouldn't require this in the config file (though we should have it so people can override) and instead just use the unix timestamp or something similar. In the past our makefiles and other things set this so it feels better as something automatic unless the user wants to specify an override.

@brooksmtownsend @autodidaddict any thoughts?

Reply:
Gah. This looks like something that's present in the OTP host NIF but may not be present in wash or in the wasmCloud runtime. The function is called `revision_or_iat`
```
631:pub(crate) fn revision_or_iat(rev: Option<i32>, iat: u64) -> Option<i32> {
632-    if rev.is_some() && rev.unwrap() > 0 {
633-        rev
634-    } else {
635-        Some(iat as i32)
636-    }
637-}
```

Reply:
yep this is the code that produces a revision if there wasn't one by reading the `iat` from the JWT. I'd just assumed the new Rust host did this as well.

Reply:
I think we have the correct implementation here then `Some(0)`, ideally we should get rid of the ability for that to be optional. In the Rust host we should then treat `0` or `None` revisions as an indicator to grab the `iat`

Reply:
Yep, that sounds right

Reply:
so, in terms of the `iat` fallback logic, should it be in this PR or another one. If it is the former, some pointer(s) on where to look would be great; i looked around and i saw a bunch of `unwrap_or_else_default()` for `revision` and `version` stuff in the `host` crate.

Reply:
I think in this PR we can only keep the logic to supply `Some(0)` as a default, and we'll need to transfer that revision functionality to wasmCloud. Alternatively, we could set up a nice helper in `wascap` to return the revision or the iat depending on the value. So @ahmedtadde I think there's nothing else to do here for wash.